cmd/brew-bootstrap-rbenv-ruby

```diff
@@ -49,9 +49,10 @@ if ! rbenv version-name &>/dev/null; then
   export RUBY_CONFIGURE_OPTS="--with-openssl-dir=$HOMEBREW_PREFIX/opt/openssl"
 
   rbenv install --skip-existing "$RUBY_DEFINITION"
-  if [ "$(rbenv exec ruby --version)" != "$(ruby --version)" ]; then
-    abort 'Error: add `eval "$(rbenv init -)"` to the end of your .bash_profile/.zshrc!'
-  fi
+fi
+
+if [ "$(rbenv exec ruby --version)" != "$(ruby --version)" ]; then
+  abort 'Error: add `eval "$(rbenv init -)"` to the end of your .bash_profile/.zshrc!'
```

Comment:
Maybe it's worth running `rbenv init` for the user and installing bundler before notifying them they need to do this? That way the script itself could run to the end, even if the user still has to take action after.